packages/drivers/local-driver/src/localDocumentStorageService.ts

```diff
@@ -114,7 +114,7 @@ export class LocalDocumentStorageService implements IDocumentStorageService {
 
 		const metadataString = IsoBuffer.from(blobContents.get(".metadata")).toString("utf-8");
 		const metadata = JSON.parse(metadataString);
-		const sequenceNumber: number = metadata.message.sequenceNumber;
+		const sequenceNumber: number = metadata.message?.sequenceNumber ?? 0;
```

Comment:
local server does not track sequence number of the snapshot? This seems weird. 
How runtime encodes metadata is runtime-specific, and could chance over time - it would be great to service (even local one) not depend on it.

Reply:
From my code tracing, it doesn't, but I didn't want to spend the time to figure out how exactly to build out that infrastructure, as it seemed a little overkill for what I needed for.
https://dev.azure.com/fluidframework/internal/_workitems/edit/7405